feat: add beta-4 target, and point --testnet to beta-4 network for forc-deploy (#4974)
FuelLabs/sway: docs/book/src/forc/plugins/forc_client/index.md

## Interacting with the testnet

To interact with the latest testnet you can pass `--testnet` flag. Passing this flag will send transactions crafted by `forc-deploy` to `beta-4` testnet.

```sh
forc-deploy --testnet
```

> Also it is possible to pass exact node url while using `forc-deploy` or `forc-run` which can be done using `--node-url` flag.

Reviewer: ```suggestion
It is also possible to pass the exact node url while using `forc-deploy` or `forc-run` which can be done using `--node-url` flag.
```

```sh
forc-deploy --node-url https://beta-3.fuel.network/graphql
```

> Finally you have `--target` option, which provides useful aliases to old testnets. For example if you want to deploy to `beta-3` you can use:

Reviewer: ```suggestion
Another alternative is the `--target` option, which provides useful aliases to all targets. For example if you want to deploy to `beta-3` you can use:
```
